lib/Screens/OpenSource.dart

```diff
@@ -124,12 +126,17 @@ class _OpenSourceState extends State<OpenSource> {
                                       ),
                                       ElevatedButton(
                                           onPressed: () async {
-                                            final url = showData[index]['link'];
-                                            if (await canLaunch(url)) {
-                                              await launch(url);
-                                            } else {
-                                              throw 'Could not launch $url';
-                                            }
+                                            Navigator.push(
+                                              context,
+                                              MaterialPageRoute(
+                                                builder: (context) =>
+                                                    WebViewWidget(
+                                                      title: showData[index]
+                                                      ['head'],
```

Comment:
Add error handlling too if you can for like before, what if the page doesn't load

Reply:
![image](https://user-images.githubusercontent.com/16136188/185358629-c766788a-91fb-4825-baa7-2921ed014571.png)
added this